CHANGELOG.md

```diff
@@ -275,7 +275,7 @@ With this change:
   - ✅ `return { data: "whatever" };`
 - If you _were_ returning a custom `status` or `headers` via `json`/`defer`:
   - We've added a new API-compatible [`unstable_data`](https://remix.run/docs/utils/data) utility that will let you send back `status`/`headers` alongside your raw data without having to encode it into a `Response`
-- We will be removing both `json` and `defer` in the next major version, but both _should_ still work in Single Fetch in v2 to allow for incremental adoption of the new behavior
+- We will be removing both `json` and `defer` in the remix major version, but both _should_ still work in Single Fetch in v2 to allow for incremental adoption of the new behavior
```

Comment:
That looks like a search and replace mistake

Reply:
no this looks right to me, the remix major version